apina-core/src/test/kotlin/fi/evident/apina/spring/testclasses/ClassWithValueClasses.kt

```diff
@@ -0,0 +1,17 @@
+package fi.evident.apina.spring.testclasses
+
+@JvmInline
+value class ValueString(private val string: String)
+
+@JvmInline
+value class ValueInteger(private val integer: Int)
+
+class ClassWithValueClasses {
+    var valueString = ValueString("test")
+    var valueInteger = ValueInteger(2)
+
+    private val privateValueString = ValueString("test")
+
+    fun getPrivateValue() = privateValueString
```

Comment:
Its a bit unclear if we are supposed to test the private field value or the getter? The name is probably misleading, since I think the code ends up analyzing the getter. If thats the intention, why not get rid of the field completely and rename the method:

```kotlin
fun getGetterValue() = ValueString("test")
```